docs/pulplift.md

```diff
@@ -91,6 +91,31 @@ If using libvirt, the [vagrant-sshfs](https://github.com/dustymabe/vagrant-sshfs
 vagrant up pulp3-source-centos7
 ```
 
+#### FIPS box pair
+
+The following boxes are meant to be run together as a pair:
+- pulp2-nightly-pulp3-source-fips-a (Pulp 3 FIPS **VM**)
+- pulp2-nightly-pulp3-source-fips-b (Pulp 2 FIPS **container** that runs **on top** of the "a" VM)
```

Comment:
not sure if this is the right place to document it, it would go to https://pulp-installer.readthedocs.io/en/latest/pulplift/
and we are avoiding claiming FIPS support, @bmbouter wdyt?

Reply:
IMO, it's ok, we are still not claiming FIPS support.
To be on the safe side to avoid potential confusion, we can leave a comment like `FIPS is not supported by Pulp, boxes are provided for those who would like test their plugins in FIPS environment and potentially apply their own FIPS patches.`
What do you all think?

Reply:
+1 from me. I might just say something simple like these boxes are for development/testing purposes only.

Reply:
+1 to claiming that these boxes are for development/testing purposes only.

Reply:
Added:

> NOTE: FIPS is not supported by Pulp. These FIPS boxes, like all the other boxes with "fips" in their
> name, are are provided for those who would like test their plugins in FIPS environment and
> potentially apply their own FIPS patches to Django.